From 0733ea11cda459116129bbd503aa56a0591060e2 Mon Sep 17 00:00:00 2001
From: Brian Bockelman <bbockelman@morgridge.org>
Date: Wed, 30 Apr 2025 08:42:31 -0500
Subject: [PATCH] CMake changes for XrdThrottle overhaul

This is done as a separate commit because upstream has rewritten the
CMake code in `master`, breaking the cherry-picking of patches back
to v5.8.1.
---
 src/XrdPlugins.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/XrdPlugins.cmake b/src/XrdPlugins.cmake
index 6abce0700..6250b4e23 100644
--- a/src/XrdPlugins.cmake
+++ b/src/XrdPlugins.cmake
@@ -121,7 +121,9 @@ add_library(
   ${LIB_XRD_THROTTLE}
   MODULE
   XrdOfs/XrdOfsFS.cc
+  XrdThrottle/XrdOssThrottleFile.cc
   XrdThrottle/XrdThrottle.hh           XrdThrottle/XrdThrottleTrace.hh
+  XrdThrottle/XrdThrottleConfig.cc     XrdThrottle/XrdThrottleConfig.hh
   XrdThrottle/XrdThrottleFileSystem.cc
   XrdThrottle/XrdThrottleFileSystemConfig.cc
   XrdThrottle/XrdThrottleFile.cc
-- 
2.43.5

